### ISSUE TITLE
Move CMO content into Draft folder for content QA/UAT

### ISSUE THREAD

**Issue by avanishgandhi** [adobe/theblog#444]
@kptdobe - Blog authors have been testing the new CMO content on oproma-theblog URL but we are seeing issues with missing authors and promos. 

**Requirement**
- need to move the EN articles in our **theblog->en->drafts->cmo** folder so QA/UAT team can test the blogs with all authors and promo content showing up correctly
- need to move the DE articles in our **theblog->de->drafts->cmo** folder so QA/UAT team can test the blogs with all authors and promo content showing up correctly

- This excel sheet is whats being used to track QA testing - https://adobe-my.sharepoint.com/:x:/r/personal/acapt_adobe_com/_layouts/15/Doc.aspx?sourcedoc=%7B89EF4349-0434-497C-AD9D-1E0A754E0105%7D&file=cmo-to-helix-pages.xlsx&action=default&mobileredirect=true

cc @rofe

**Comment by kptdobe** [comment]
ok. I can move the articles in the corresponding folders tomorrow. Here is the plan:

- move article under /LANG/drafts/cmo
- copy authors into /LANG/authors - do not overwrite existing ones (to keep the existing avatar and bios)
- I assume there are no new promotions to copy

**Comment by avanishgandhi** [comment]
That works.

thanks @kptdobe

**Comment by kptdobe** [comment]
I started to look at the content. I copied the articles into the appropriate folders. But...

- authors should have worked on the oproma instance. There are few import issues in the author file names, I sent a mail to @christianleger to address the issues during the import. After that, any author issue is a bug to look at.
- same for promotions, except that originals should have been copied from `theblog/en/promotions` to the `oproma/en/promotions` folder.
